client_sdk/go/fpc/resmgmt/lifecycleclient_test.go

```diff
@@ -0,0 +1,7 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package resmgmt
```

Comment:
I guess a '// TODO: Implement me' is missing here (or we could have left out the file completely?)

Reply:
Added the TODO. I think having this file with a TODO produces more pressure also to write this test code!